server/cpho/queries.py

```diff
@@ -81,3 +81,11 @@ def get_submission_statuses(indicator, period):
         "statuses_by_dimension_type_id": submission_status_by_dimension_type_id,
         "global_status": global_status,
     }
+
+
+def relevant_dimension_types_for_period(indicator, period):
+    data = indicator.data.filter(period=period).distinct("dimension_type")
+    dimension_types = set([d.dimension_type for d in data]).union(
+        indicator.relevant_dimensions.all()
+    )
```

Comment:
I think we should have a `prefetch_related('dimension_type')` on the first query? If it doesn't play well with the `distinct()`, it's not a big deal because we're putting it through a `set()` anyway